pkgs/development/python-modules/numba/default.nix

```diff
@@ -27,17 +27,16 @@
 let
   inherit (cudaPackages) cudatoolkit;
 in buildPythonPackage rec {
-  # Using an untagged version, with numpy 1.25 support, when it's released
-  # also drop the versioneer patch in postPatch
```

Comment:
we want to keep that hint, no?

Reply:
> we want to keep that hint, no?

I agree.

Reply:
Sorry must have gotten removed when I resolved merge conflicts. I'll update.